src/waltham/waltham-connection.c

```diff
@@ -534,7 +534,7 @@ wth_connection_set_registry_callback(struct wth_connection *conn,
 	conn->registry_callback = callback;
 }
 
-WTH_EXPORT void
+void
 wth_connection_send_error(struct wth_connection *conn, struct wth_object *obj,
```

Comment:
Also needs the declaration moved from the public header to a private header.

Reply:
How would you name this header, waltham-connection-private.h?

Reply:
`waltham-private.h` seems appropriate.